### DISCUSSION TITLE
Firmware Upgrade and Release Note | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#903] [Q&A]
# Firmware Upgrade and Release Note | Seeed Studio Wiki

SenseCAP T1000-A/B Tracker Firmware Release Note

https://wiki.seeedstudio.com/fm_release_for_SenseCAP_T1000/

**Comment by Ecofuel06** [comment]
Hi, the Firmwareupdate from 2.2 to 2.5 was not successfull. Now i can´t make the Firmwareupdate per Bluetooth...where can i find the Firmwarefile? Please send me an Link to h-wilmink@hotmail.com

**Comment by Ecofuel06** [comment]
O.K. Sorry its a bad service...

here you can find the Firmware 2.5....

https://forum.seeedstudio.com/t/sensecap-t1000-gps-tracker-failed-firmware-upgrade-now-stuck-with-blinking-led/273782/7

**Comment by lazyzero** [comment]
I use two T1000-B as trackers to map the The Things Network coverage in my region. And I observe strange packets. Both are set to not cache geolocation. But when the trackers are close to a gateway they send some days 1-6 packets with "historical" location data that jeopardises the mapping data. I am on firmware 2.5.

BTW is the GPS time also in the payload?

Any idea or a firmware version that is not prone by this bug?

**Reply by nfs0619** [reply]
Thanks for the detailed info! For issues like unexpected historical location data, it might be best to contact our technical support team directly so they can assist you further — you can reach them at **[sensecap@seeed.cc](mailto:sensecap@seeed.cc)**.

Alternatively, feel free to post your question on our community forum: [https://forum.seeedstudio.com/](https://forum.seeedstudio.com/) — the team and other users are active there and may offer additional insights.

**Comment by impromatch** [comment]
Dear Seeed Studio Technical Support,

We are currently using SenseCAP T1000-A model devices in our LoRaWAN-based tracking project. However, we have noticed that the GPS-based location data (latitude and longitude) is always returning as 0 in the uplink payload, despite the device being positioned outdoors with a clear view of the sky.

Here is a sample uplink message received from the device:

{
  "applicationID": "2",
  "applicationName": "test mqtt",
  "data": "BVkBBwE=",
  "devEUI": "2**************a",
  "deviceName": "beyaz-cihaz",
  "fCnt": 699,
  "fPort": 5,
  "gatewayTime": "2025-08-08T09:18:03+03:00",
  "rxInfo": [{
    "altitude": 0,
    "latitude": 0,
    "longitude": 0,
    "rssi": -48,
    "loRaSNR": 13.8
  }],
  "txInfo": {
    "frequency": 868300000
  }
}

**Reply by Kian-kang** [reply]
Please export a segment of data and send it to our SenseCAP technical support email address: sensecap@seeed.cc

**Comment by osarbishei** [comment]
We are using T1000-B devices running Firmware v2.5 to conduct some experiments collecting GPS data using LoRa. We have our own LoRa gateway that collects this data. We did not find any settings in the SenseCraft mobile app within the Bluetooth configuration section to set the radio transmission power for LoRa transmission. Therefore, we were wondering what is the LoRa radio transmission power hardcoded to in firmware v2.5? Is it hardcoded to 0dB or is it set to the maximum radio power allowed for each region? Our region is US915. We are doing some range testing with our own gateway, and we believe that the LoRa radio transmission power is not set to the maximum value allowed for our region US915 on the sensor, which is +30dB. I would hope that we could have the option to increase the Tx power on the sensor.

**Reply by osarbishei** [reply]
To add a bit more context, we use these settings for the sensor in the SenseCraft mobile app to increase the coverage range in our experiments:

**Reply by osarbishei** [reply]
ADR: disabled (no automatic data rate selected)
DR1: Data Rate 1 @125kbps
Protocol: custom protocol
Subband: 2
data caching: disabled
region: US915
Spreading Factor (SF): 9

**Reply by nfs0619** [reply]
Hi, thank you for your feedback!
Currently, the LoRa transmission power on our devices is fixed and cannot be modified.

* On high-power channels, the Tx power is 19–21 dBm
* On low-power channels, the Tx power is 12–14 dBm

**Comment by GarnetNR** [comment]
At the top of this page it shows a screen capture for "Upgrade by Cloud", but no instructions on how to navigate to this location and select the "Device Bluetooth Configuration" option.  What am I missing?

**Reply by nfs0619** [reply]
Hello, please follow this tutorial: https://wiki.seeedstudio.com/Get_Started_with_SenseCAP_T1000_tracker/#get-started to scan the QR code and download our **SenseCraft App** first.
